packages/compartment-mapper/src/parse-pre-cjs.js

```diff
@@ -50,6 +52,7 @@ export const parsePreCjs = async (
       reexports,
       exports,
       execute,
+      __syncModuleProgram__: syncModuleProgram,
```

Comment:
note: Name it `cjsFunctor` instead, so that it doesn't trigger SES to worry about it